concrete/src/Notification/Alert/Filter/FilterInterface.php

```diff
@@ -0,0 +1,13 @@
+<?php
+namespace Concrete\Core\Notification\Alert\Filter;
+
+use Concrete\Core\Notification\Alert\AlertList;
+
+interface FilterInterface
+{
+
+    public function getName();
+    public function getKey();
+    public function filterAlertList(AlertList $list);
```

Comment:
These methods could use comments that make it clear what the input/output is and what they do

Reply:
👍